src/jasmineTestRunner/JasmineReporter.js

```diff
@@ -106,7 +106,7 @@ function (container, ancestorTitles, spec) {
           // have to regexp out the message from the stack string in order to
           // colorize the `message` value
           result.trace.stack = result.trace.stack.replace(
-            /(^.*$(?=\n\s*at))/m,
+            /(^(.|\n)*?(?=\n\s*at\s))/,
```

Comment:
- Removed the multiline flag, as it causes `^` to match the beginning of *any* line, and javascript's regexps don't support `\A`
- Rewrote `.` as `(.|\n)` because we disabled the multiline flag.
- changed `*` to `*?`, making it non-greedy. We want to consume from the top to the *first* copy of `at `, not the last one.
- `$` isn't needed because of the lookahead that matches `\n`
- Put `\s` after `at` because it allow us to avoid catching some error messages that might start with `at` on a line. See our unit test where we use `attention` as an object key. (still not bulletproof, but marginally better)